### PR TITLE
Add integration test for compiled js bundle size

### PR DESCRIPTION
**Pull request by perclasson**
Closes https://github.com/flutter/gallery/issues/101

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -62,3 +62,35 @@ jobs:
...
+    # Clone the Gallery repository under `code`, to avoid conflicts with `flutter`.
+    - name: Clone the Gallery repository.
+      uses: actions/checkout@v2
+      with:
+        path: 'code'
+
+    - name: Get packages for the Flutter project.
+      run: flutter pub get
+      working-directory: code
+    - name: Run Flutter integration tests.
```
Comment (guidezpl) [inline .github/workflows/main.yml:94]: Could you rename this to 'Run gallery_compile_test' with a command for that test specifically? That way when we add more tests it'll be easier to view their outputs separately in the Github run result.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -62,3 +62,35 @@ jobs:
       # TODO: Enable verify-code-segments for Unix and Windows when it behaves the same as on macOS:
       # https://github.com/material-components/material-components-flutter-gallery/issues/565
       if: runner.os == 'macOS'
+
+  integration-test:
+    name: Integration test
```
Comment (guidezpl) [inline .github/workflows/main.yml:67]: nit: Integration tests

File: .github/workflows/main.yml (hunk 1 above)
Comment (perclasson) [reply .github/workflows/main.yml:94]: That makes sense, will change it to that

File: .github/workflows/main.yml (hunk 2 above)
Comment (perclasson) [reply .github/workflows/main.yml:67]: Done!

**Pull request merged**